plugins/source/k8s/client/helpers.go

```diff
@@ -52,34 +52,85 @@ func ResolveContext(_ context.Context, meta schema.ClientMeta, r *schema.Resourc
 	return r.Set(c.Name, client.Context)
 }
 
-// In k8s, IP Addresses may sometimes be empty-strings - but postgresql doesn't like that.
-// So, the resolver for ip-addresses should recognize that case and not set null instead.
+// In k8s, IP Addresses may sometimes be empty-strings or `None` - but postgresql doesn't like that.
+// So, the resolver for ip-addresses should recognize that case and set null instead.
 func StringToInetPathResolver(path string) schema.ColumnResolver {
 	return func(_ context.Context, meta schema.ClientMeta, r *schema.Resource, c schema.Column) error {
 		value := funk.Get(r.Item, path, funk.WithAllowZero())
 
 		stringValue, ok := value.(string)
-		if ok && stringValue != "" {
+		if ok && stringValue != "" && stringValue != "None" {
 			return r.Set(c.Name, value)
 		}
+
 		return r.Set(c.Name, nil)
 	}
 }
 
-// In k8s, IP Addresses may sometimes be empty-strings - but postgresql doesn't like that.
-// So, the resolver for ip-addresses should recognize that case and not set null instead.
+// In k8s, IP Addresses may sometimes be empty-strings or `None` - but postgresql doesn't like that.
+// So, the resolver for ip-addresses should recognize that case and set null instead.
+func StringToInetArrayPathResolver(path string) schema.ColumnResolver {
+	return func(_ context.Context, meta schema.ClientMeta, r *schema.Resource, c schema.Column) error {
+		value := funk.Get(r.Item, path, funk.WithAllowZero())
+
+		stringArrayValue, ok := value.([]string)
+		if !ok {
+			return r.Set(c.Name, nil)
+		}
+
+		sanitized := []*string{}
+
+		for i := range stringArrayValue {
+			if stringArrayValue[i] != "" && stringArrayValue[i] != "None" {
+				sanitized = append(sanitized, &stringArrayValue[i])
+			} else {
+				sanitized = append(sanitized, nil)
+			}
+		}
```

Comment:
Actually, it could be fixed even easier: https://go.dev/play/p/p6YgBjZXixI
```suggestion
		for _, v := range stringArrayValue {
			v := v
			if v == "" || v == "None" {
				sanitized = append(sanitized, nil)
				continue
			}
			sanitized = append(sanitized, &v)
		}
```